rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -5,16 +5,20 @@ import com.google.protobuf.ByteString
 import coop.rchain.models.{PCost, Par, ProtoM, StacksafeMessage}
 
 //TODO(mateusz.gorski): Adjust the costs of operations
-final case class Cost(value: Long) extends AnyVal {
-  def *(base: Int): Cost   = Cost(value * base)
-  def +(other: Cost): Cost = Cost(value + other.value)
-  def -(other: Cost): Cost = Cost(value - other.value)
+final case class Cost(value: Long, operation: String) {
+  def *(base: Int): Cost   = Cost(value * base, s"$operation * $base")
+  def +(other: Cost): Cost = Cost(value + other.value, "")
+  def -(other: Cost): Cost = Cost(value - other.value, "")
```

Comment:
discarding the other's 'operation' seems wrong in here

Reply:
Please include the `other` oepration name for + and -

Reply:
This change will have to be reverted, otherwise the heap will run out of space